qemu/tests/win_virtio_update.py

```diff
@@ -36,6 +36,17 @@ def check_cdrom(timeout):
                 break
         return vols
 
+    def virtio_driver_install(session, cmd, driver):
```

Comment:
@vipmike007 

Could you add some comments to introduce the func() ?

Thanks.